float64 a
float64 b
float64 alpha
---
bool success
string msg
